utils/converters/fbx2three.js

```diff
@@ -0,0 +1,80 @@
+var fs = require( 'fs' );
+var path = require( 'path' );
+
+if ( process.argv.length <= 2 ) {
+
+	console.log( `Usage: ${path.basename( __filename )} model.fbx` );
+	process.exit( - 1 );
+
+}
+
+//
+
+var PRECISION = 6;
+
+function parseNumber( key, value ) {
+
+	return typeof value === 'number' ? parseFloat( value.toFixed( PRECISION ) ) : value;
+
+}
+
+THREE = require( '../../build/three.js' );
+require( '../../examples/js/curves/NURBSCurve.js' );
+require( '../../examples/js/curves/NURBSUtils.js' );
+require( '../../examples/js/loaders/FBXLoader.js' );
+global.Zlib = require( '../../examples/js/libs/inflate.min.js' ).Zlib;
+
+global.window = {
+	innerWidth: 1024,
+	innerHeight: 768,
+	URL: {
+		createObjectURL: function () {
+			throw new Error( 'fbx2three: Images in binary format not yet supported.' );
+		}
+	}
+};
+
+// HTML Images are not available, so use a Buffer instead.
+THREE.ImageLoader.prototype.load = function ( url, onLoad ) {
+
+	if ( this.path !== undefined ) url = this.path + url;
+
+	// If image isn't found, try to ignore it.
+	if ( !fs.existsSync( url ) ) {
+
+		onLoad( new Buffer( '' ) );
+		return;
+
+	}
+
+	onLoad( fs.readFileSync( url ) );
+
+};
+
+// Convert image buffer to data URL.
+THREE.Texture.prototype._getDataURL = function ( image ) {
```

Comment:
Would it make any sense to add `Buffer` type support to `getDataUrl()` in Texture.js?

Reply:
If this part worked alone I’d say yes, but it relies on ImageLoader returning a Buffer instead of an Image and that part feels more fragile.. no great cross-environment way to represent an image that I know of. A full nodejs rewrite of the ImageLoader/TextureLoader/Texture classes would solve this, but probably overkill for this converter.